lib/scripts/lingui.ts

```diff
@@ -0,0 +1,24 @@
+// Wrapper around lingui command that exits with a non zero value if an error is detected
+import * as child from "child_process";
+
+const args: string[] = process.argv.splice(2);
+const command = `NODE_ENV=test npx lingui ${args.join(" ")}`;
+child.exec(command, (error, stdout, stderr) => {
+  if (error) {
+    // Handle exec command error (lingui command not found?)
+    console.log(`error: ${error}`);
+    throw error;
+  }
+  if (stderr) {
+    // Display command standard error
+    console.log(`stderr: ${stderr}`);
+    // This signals that something wrong happened during the command (BEWARE OF CHANGES IN LINGUI OUTPUT)
+    if (stderr.includes("Cannot process file")) {
+      process.exit(1);
+    }
+    return;
+  }
+  // Display command standard output
+  console.log(`stdout: ${stdout}`);
```

Comment:
This line is never reached.
There is always a `stderr` at least on my machine which outputs 
"(use "npm run lingui:extract:dev" to update catalogs with new messages)
(use "npm run lingui:compile" to compile catalogs for production)"

I would console log the standard stdout at the beginning of the function.
Or you remove the return in line 19.

Reply:
Good catch!